test/NosCore.Tests/PacketGenerationTests.cs

```diff
@@ -32,11 +34,11 @@ public void Setup()
         [TestMethod]
         public void GenerateInPacketIsNotCorruptedForCharacter()
         {
-            var characterTest = new Character() { Name = "characterTest" };
+            var characterTest = new Character() { Name = "characterTest", Account = new AccountDTO {Authority = AuthorityType.Administrator}, Level = 1};
 
             var packet = PacketFactory.Serialize(new[] { characterTest.GenerateIn() });
             Assert.AreEqual(
-                "in 1 characterTest - 0 0 0 0 0 0 0 0 0 -1.-1.-1.-1.-1.-1.-1.-1.-1 0 0 0 -1 0 0 0 0 0 0 0 0 0 - 0 0 0 0 0 0 0 0 0 0 0",
+                "in 1 characterTest - 0 0 0 0 3 0 0 0 0 -1.-1.-1.-1.-1.-1.-1.-1.-1 0 0 0 -1 0 0 0 0 0 0 0 0 -1 - 1 0 0 0 0 1 0 0 0 0 0",
```

Comment:
use (byte)Authority instead of 3 it will be easier to test. same for level to be sure that the level is at the correct place :)